css/base.css

```diff
@@ -0,0 +1,3 @@
+body {
+    font-family: 'Open Sans', sans-serif;
+}
```

Comment:
New line at end of file 💅 
Dá pra configurar no editor como padrão, aí ele já para de apitar aqui no github 😊